MongoDB.Analyzer.sln

```diff
@@ -9,8 +9,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MongoDB.Analyzer.Package",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MongoDB.Analyzer.Tests", "tests\MongoDB.Analyzer.Tests\MongoDB.Analyzer.Tests.csproj", "{ACB908A0-7F5C-44DA-B9D2-CCFBCAEADD1D}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MongoDB.Analyzer.Vsix", "src\MongoDB.Analyzer.Vsix\MongoDB.Analyzer.Vsix.csproj", "{E131D033-9174-4B75-AD2F-3EA0FF390B7F}"
```

Comment:
Are any changes needed in the solution for this PR? Probably it should be reverted.

Reply:
Reverted.